src/com/goide/runconfig/testing/frameworks/gocheck/GocheckRunConfigurationProducer.java

```diff
@@ -43,11 +43,6 @@ protected String getFunctionConfigurationName(@NotNull GoFunctionOrMethodDeclara
            : super.getFunctionConfigurationName(function, fileName);
   }
 
-  @Override
-  protected boolean shouldSkipContext(@Nullable GoFunctionOrMethodDeclaration context) {
```

Comment:
This producer still should ignore GoFunctionDeclarations but now it can do it overriding `findTestFunctionInContext` method